### ISSUE TITLE
MDT doesn't display properly at lower resolutions

### ISSUE THREAD

**Issue by RockemRobotics** [fkk-cz/noire-public#542]
This issue doesn't affect most folks, but anyone playing on a laptop may have this issue. The MDT does not display properly unless there are at least 1600 pixels horizontal and 1024 pixels vertical. The information is all there, but the UI causes much of the information to be pushed below the bottom edge of the MDT window, and sometimes the bottom edge of the screen as well. This is most notable on 1366x768 16:9 screens.

![MDT Screenshot](https://user-images.githubusercontent.com/14297779/151620151-b2672e7d-c89c-4ad7-afe4-73dd3451cc0d.png)

**Comment by No-Yes1** [comment]
Can you try something for me? Go to settings - system - display - "change the size of text, apps and other items" - see if it's on 100%?

**Comment by RockemRobotics** [comment]
> Can you try something for me? Go to settings - system - display - "change the size of text, apps and other items" - see if it's on 100%?

I actually did this test by just adjusting the resolution in Advanced Graphics - Resolution without changing display scaling from 100%. That's how I came up with the numbers 1600 for width and 1024 for height.

A friend in the DOJ has a laptop monitor at 1366 x 768 resolution and has this issue, which is why I investigated. He can increase the resolution in Advanced Graphics but it then scales everything else down and makes it harder to read the text.
